django_cloud_tasks/apps.py

```diff
@@ -104,13 +103,29 @@ def get_task_metadata_class(self):
 
         return metadata_class
 
-    def _fetch_config(self, name: str, default: Any, as_list: bool = False) -> Any:
+    def _fetch_config(self, name: str, default: Any) -> Any:
         config_name = f"{PREFIX}{name.upper()}"
+        return getattr(settings, config_name, os.environ.get(config_name, default))
 
-        value = getattr(settings, config_name, os.environ.get(config_name, default))
-        if as_list and not isinstance(value, list):
-            value = value.split(",")
-        return value
+    def get_str(self, name: str, default: Any) -> str:
```

Comment:
The dedicated method are a great addition, but the naming "get_type" is too generic for a method inside an App class.
 
```suggestion
    def _fetch_str_config(self, name: str, default: Any) -> str:
```

Reply:
done